Server/config/config.js

```diff
@@ -18,10 +16,7 @@ module.exports = {
     dialect: 'postgres',
   },
   production: {
-    username: 'xcnwqlat',
-    password: 'zmYX6GdPo_2tzznPiKoYSCbfPXp_Cci7',
-    database: 'xcnwqlat',
-    host: 'baasu.db.elephantsql.com',
-    dialect: 'postgres',
+      use_env_variable: process.env.DATABASE_URL,
```

Comment:
Can you use make your environment variable more descriptive. I expect you to have an environment variable for production and test.